A72-open-telemetry-tracing.md

```diff
@@ -0,0 +1,560 @@
+The existing gRPC OpenCensus tracing APIs in grpc-census plugin are different between
+languages, e.g. in grpc-java it is zero-configuration: as long as the grpc-census
+dependency exists in the classpath, the traces are automatically generated.
+In C++, there is an API for users to call to enable tracing. In Go, it is exposed via stream tracers.
+We keep the grpc-census plugin to allow users who already depend on grpc-census to
+continue using it for newer grpc versions.
+
+gRPC users depending on the grpc-census plugin have non-trivial migration paths
+to OpenTelemetry. Consider the following use cases:
+1. Compatibility between a gRPC client and server as two distributed components,
+where during migration one will use OpenCensus and the other will use OpenTelemetry. 
+2. Migrate an application binary where both OpenCensus and OpenTelemetry maybe exist
+   in the dependency tree. This can be the application’s own tracing code, or gRPC
+   OpenCensus, or other dependencies that involve OpenCensus and/or OpenTelemetry.
+
+Here are the suggested solutions for both use cases. 
+
+### Migrate to OpenTelemetry: Cross-process Networking Concerns
+When users first introduce gRPC OpenTelemetry, for the time window when the
+gRPC client and server have mixed plugins of OpenTelemetry and OpenCensus, 
+it is encouraged to use `GrpcTraceBinPropagator`, as described in the previous section,
+that propagates `grpc-trace-bin` header for the migration.
+When using `grpc-trace-bin` the OpenCensus spanContext and
+OpenTelemetry spanContext are identical, therefore a gRPC OpenCensus client can
+speak with a gRPC OpenTelemetry server and vice versa. Using the same header 
+greatly simplifies rollout.
+
+After migration period, users have the flexibility to switch to other propagators.
+OpenTelemetry and its extension packages support multiple text map propagators. 
+The gRPC OpenTelemetry API allows specifying 
+multiple propagators: either public standard ones or custom propagators that 
+implement the OpenTelemetry propagator API interface. The API composites the 
+propagators and gRPC puts all the propagator data into the wire through metadata. 
+This allows users to easily migrate a group of applications with an old propagator to 
+a new propagator. An example migration path can be:
+1. Configure server to accept both old and new propagators.
+2. Configure the client with the desired new propagators and to drop the old propagator.
+3. Make the server only accept the new propagators and complete the migration.
+
+### Migrate to OpenTelemetry: In Binary
+The OpenCensus [shim](https://github.com/open-telemetry/opentelemetry-java/tree/main/opencensus-shim)
+(currently available in Java, Go, Python) allows binaries that have a mix of 
+OpenTelemetry and OpenCensus dependencies to export trace spans from both 
+frameworks, and keep the correct parent-child relationship. This is the 
+recommended approach to migrate to OpenTelemetry in one binary gradually. 
```

Comment:
Instead of saying "in one binary gradually", how about we say "incrementally within a single binary"?